features/bootstrap/ApiContext.php

```diff
@@ -183,6 +183,15 @@ public function assertResponseGrid(): void
         $this->assertJsonObjectContainsKeys('configuration,columns,collection,info');
     }
 
+    /**
+     * @Then /^print last api response$/
+     */
+    public function printLastApiResponse(): void
```

Comment:
This method will be for debug?

Reply:
Yes it is for debug.

Reply:
Ok we can create ApiDebugContext, but if we have to move more than this one method, then we can crate a new task for this.